src/redux/actions/registration.action.js

```diff
@@ -0,0 +1,29 @@
+import {
+  REGISTER_REQUEST_SUCCEEDED,
```

Comment:
What is the difference between registration action and registration action creator?

Reply:
I don't know...
I merged the 2 files into one file and deleted the registration action creator...